src/5e-SRD-StartingEquipment.json

```diff
@@ -4,844 +4,678 @@
-			"item": { "url": "/api/equipment/greatclub", "name": "Greatclub"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/handaxe", "name": "Handaxe"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/javelin", "name": "Javelin"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/light-hammer", "name": "Light hammer"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/mace", "name": "Mace"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/quarterstaff", "name": "Quarterstaff"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sickle", "name": "Sickle"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/spear", "name": "Spear"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/crossbow-light", "name": "Crossbow, light"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/dart", "name": "Dart"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/shortbow", "name": "Shortbow"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sling", "name": "Sling"}, "quantity": 1}]
-	}],
-	"choice_2": [{
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/component-pouch", "name": "Component pouch"}, "quantity": 2}]
-	}, {
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/crystal", "name": "Crystal"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/orb", "name": "Orb"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/rod", "name": "Rod"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/staff", "name": "Staff"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/wand", "name": "Wand"}, "quantity": 1}]
-	}],
-	"choice_3": [{
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/dungeoneers-pack", "name": "Dungeoneer's Pack"}, "quantity": 1}]
-	}, {
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/explorers-pack", "name": "Explorer's Pack"}, "quantity": 1}]
-	}],
+	"starting_equipment": [
+		{ "equipment": { "url": "/api/equipment/dagger", "name": "Dagger" }, "quantity": 2 }
+	],
+	"starting_equipment_options": [
+		{
+			"choose": 1,
+			"type": "equipment",
+			"from": [
+				[
+					{ "equipment": { "url": "/api/equipment/crossbow-light", "name": "Crossbow, light" }, "quantity": 1 },
+					{ "equipment": { "url": "/api/equipment/crossbow-bolt", "name": "Crossbow bolt" }, "quantity": 20 }
+				], {
+					"equipment_option": {
+						"choose": 1,
+						"type": "equipment",
+						"from": {
+							"equipment_category": {
+								"name": "Simple Weapons",
+								"url": "/api/equipment-categories/simple-weapons"
+							}
+						}
+					}
+				}
+			]
+		}, {
+			"choose": 1,
+			"type": "equipment",
+			"from": [
+				{ "equipment": { "url": "/api/equipment/component-pouch", "name": "Component pouch" }, "quantity": 2 },
+				{
+					"equipment_option": {
```

Comment:
Do we need this or can it be combined into the parent choice? I think this is also in a few other places too.

Reply:
Huh.... in the SRD, it's just `Arcane Focus` and not all of these choices. Weird. That looks like it was weirdly pulled from the SRD:
```
An arcane focus is a special item—an orb, a crystal, a rod, a specially constructed staff, a wand-like length of wood, or some similar item—designed to channel the power of arcane spells. A sorcerer, warlock, or wizard can use such an item as a spellcasting focus.
```

Reply:
Huh: https://github.com/bagelbits/5e-database/blob/master/src/5e-SRD-Equipment.json#L1924-L2012

Reply:
Don't worry about fixing the Arcane Focus issue. That should be a separate PR.

Reply:
I'll throw one together once this gets merged in.

Reply:
When applying my initial changes, I did wonder whether to combine these. I chose to leave them as-is, so as to better reflect the source text of the SRD, as well has the Player's Handbook. It would certainly make sense to merge them in order to reduce the number of choices a user has to make, but I wonder if that should be prioritised over aligning with the source texts, for familiarity's sake. Thoughts?

Reply:
Ah alright. Will leave it for now.

Reply:
Hrmmmm. Okay yeah. I see the issue. Especially since each kind of Arcane Focus has a different weight and cost. Well this is weird. This might make more sense as an Equipment Category? Maybe?

Reply:
@ogregoire Thoughts?

Reply:
Arcane Focus :

> Arcane Focus. An arcane focus is a special item — an orb, a crystal, a rod, a specially constructed staff, a wand -­‐‑ like length of wood, or some similar item — designed to channel the power of arcane spells. A sorcerer, warlock, or wizard can use such an item as a spellcasting focus.

There's no other description or category, so I'd just use "Arcane Focus".

Reply:
@ogregoire 
The catch is that arcane focus isn't really listed in the cost weight table, instead it's orb, crystal, rod, staff, or wand. If you look on page 69 of the SRD.

Reply:
And the Equipment table has those listed as well instead of Arcane Focus.

Reply:
Worth opening a new issue/PR before this discussion gets any longer?

Reply:
My best thought is to setup an equipment category for Arcane Focus and include those items in the category instead, which more accurately encapsulates how the data is actually protrayed?

Reply:
@fergcb Yes please.

Reply:
Please link the issue here! :D

Reply:
Here's the issue: https://github.com/bagelbits/5e-database/issues/234